content/en/docs/instrumentation/rust/_index.md

```diff
@@ -23,7 +23,7 @@ as follows:
 
 ## Crates
 
-Opentelemetry for Rust contains the following crates:
+Opentelemetry for Rust published the following crates:
```

Comment:
Nits: Opentelemetry -> OpenTelemetry? And maybe "publishes" rather than "published"?